OasisDEX/oasis-borrow
# As/add mpa support to goerli

# [Integrate MPA & New DummyExchange](https://app.shortcut.com/oazo-apps/story/3199/integrate-mpa-new-dummyexchange-from-goerli-with-oasis-borrow)

## Changes 👷‍♀️
Adds adresses of MPA and Dummy Exchange 
  
## How to test 🧪
 - It should be possible to run oasis with `network=goerli`
 - You should be able to open multiply vault
 - You hould be able to adjust multiply vault

keep in mind that Dummy exchange has limitedd amount of ETH and DAI so it will work only for small enough vaults

Among other changes, I updated MakerDAO adresses from here:
https://changelog.makerdao.com/releases/mainnet/1.9.12/index.html
and here:
https://changelog.makerdao.com/releases/goerli/1.9.11/index.html

and there are differences with adresses we currently have on production. (not only new one)

probably that do not affects us since we are using Maker in limited way but is little alarming

also while updating addresses list I encountered an issue, described as separate story here:
https://app.shortcut.com/oazo-apps/story/3326/investigate-latest-makerdao-changelog
